### PR TITLE
Include private browsing as a cause for this CORS error

### PR DESCRIPTION
**Pull request by JamesJansson**

### CODE REVIEW

**Review by hamishwillee — APPROVED** [review]
Thanks. Why does this happen and in what browser? Can you provide an example I can test?

It seems quite likely that private private browser might have different security requirements that could block a CORS request, but would be good to verify.

**Comment by JamesJansson** [comment]
> Thanks. Why does this happen and in what browser? Can you provide an example I can test?

129.0 (64-bit) Mozilla Firefox Snap for Ubuntu canonical-002 - 1.0

It happened on my site, both on a dev machine on localhost and on [https://fociva.com](https://fociva.com)

If an error occurs, the app attempts to send a request to Sentry's server at https://o508829.ingest.us.sentry.io

> It seems quite likely that private private browser might have different security requirements that could block a CORS request, but would be good to verify.

I believe that is the reason. It's not a bug, but simply a reason why you might see this "error". That is, it is working as designed (even if the error message isn't clear).

File: files/en-us/web/http/cors/errors/corsdidnotsucceed/index.md [hunk 1]
```diff
@@ -14,24 +14,19 @@ Reason: CORS request did not succeed
...
-- The server did not respond to the actual request (even if it responded to the
-  {{Glossary("Preflight request")}}). One scenario might be an HTTP service being
-  developed that panicked without returning any data.
+- Trying to access an `https` resource that has an invalid certificate will cause this error.
+- Trying to access an `http` resource from a page with an `https` origin will also cause this error.
+- From Firefox 68 to Firefox 84, `https` pages were not permitted to access `http://localhost`.
+  This has been changed with [Bug 1488740](https://bugzil.la/1488740).
+- The server did not respond to the actual request (even if it responded to the {{Glossary("Preflight request")}}).
+  One scenario might be an HTTP service being developed that panicked without returning any data.
+- The window is in "Private Browsing" mode.
```
Comment (hamishwillee) [inline files/en-us/web/http/cors/errors/corsdidnotsucceed/index.md:29]: OK, I'll spell it out. This makes sense to me.

```suggestion
- The window is in "Private Browsing" mode (which may have security requirements that could block a CORS request).
```

**Review by hamishwillee — APPROVED** [review]
Thanks @JamesJansson !

**Pull request merged**